### PR TITLE
{bio}[goolf/1.7.20] IQ-TREE v1.5.6 (MPI)

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb]: @pescobar `s/hpcugent/easybuilders/g` to make Travis happy

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+easyblock = "CMakeMake"
+
+name = 'IQ-TREE'
+version = '1.5.6'
+versionsuffix = '-MPI'
+
+homepage = 'http://www.iqtree.org'
+description = "A fast and effective stochastic algorithm to infer phylogenetic trees by maximum likelihood"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
```
Comment (boegel) [inline easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb]: @pescobar No `toolchainopts = {'usempi': True}` needed?

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'IQ-TREE'
+version = '1.5.6'
+versionsuffix = '-MPI'
```
Comment (boegel) [inline easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb]: @pescobar Use `-mpi` to match with existing `IQ-TREE` easyconfig that uses `-omp-mpi` for hybrid build?

Also, is there a point to build a separate MPI vs multithreaded build, as opposed to just enabled both `omp` and `mpi` in a single build? Cfr. #3695

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb]: Also, we had to disable architecture-specific optimisations (`'optarch': False`) in #3695...

How have you built `IQ-TREE`, any related `--optarch` settings you used?

File: easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+source_urls = ['https://github.com/Cibiv/IQ-TREE/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c4441cb251081eb8ee31029bf2e75330be41f86d4dbcb59543de6c0c3d63cc83']
+
+separate_build_dir = True
+
+builddependencies = [('CMake', '3.4.3')]
+
+configopts = ' -DIQTREE_FLAGS=mpi'
```
Comment (boegel) [inline easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb]: Actually, even better: you can build all flavors in a single easyconfig:

```python
configopts = ['-DIQTREE_FLAGS=mpi', '-DIQTREE_FLAGS=omp', '-DIQTREE_FLAGS="omp mpi"']

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**